app/controllers/application_controller.rb

```diff
@@ -1,4 +1,25 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  attr_accessor :current_user
+
+  before_action :ensure_json_request
+
+  protected
+
+  ##
+  # Enforce authentication (request Authorization header should be set to a user's uuid)
+  #
+  def authenticate!
+    @current_user = User.find_by :uuid => request.headers['Authorization']
```

Comment:
Are we going to be using the old hash rocket instead of the newer updated syntax? ;)

Reply:
I prefer hash rockets instead of the newer syntax ;)